src/ConfigFile.c

```diff
@@ -438,6 +478,38 @@ void Ros_ConfigFile_CheckUsbForNewConfigFile()
         Ros_Debug_BroadcastMsg("No new configuration file found on CN102 USB drive.");
 }
 
+STATUS Ros_ConfigFile_HostOnNetworkInterface(char* const host, USHORT if_no, BOOL* const reachable)
```

Comment:
pedantically, this wouldn't belong here. It's not related to config file parsing.

We could decide to move it elsewhere.

Reply:
I can't think of a better place for it, unless we make a whole new file.